build.gradle

```diff
@@ -190,8 +190,10 @@ allprojects {
                     force "org.apache.commons:commons-math3:${commonsMath3Version}"
                     // force version for api, accounts, fileTransfer, harvest, scrumtime, docker
                     force "org.apache.httpcomponents:httpcore:${httpcoreVersion}"
+                    force "org.apache.httpcomponents.core5:httpcore5:${httpcore5Version}"
```

Comment:
Is the line above this meant to go away? (Probably in a later PR, I'm guessing?)

Reply:
Yes, a few modules (Lincs, TargetedMS, Connectors) still use the old version and they'll get migrated in a follow-on PR. I mentioned this in the platform PR, but probably should have mentioned it here.